src/v2/ion/ui-schema/ion-boolean-handler.js

```diff
@@ -32,6 +32,17 @@ const getRadioUiSchema = ({ label, required, options }) => ({
   sublabel: required? null : loc('oie.form.field.optional', 'login'),
 });
 
+const getCheckboxUiSchemaSSR = ({ label, type, required }) => ({
```

Comment:
I would suggest we rename this to make it more generic instead of tying it to a specific business flow. 
maybe `getCheckboxUiSchemaWithDefaultValue` ?

Reply:
good idea. updated